tools/launch_instance.rb

```diff
@@ -681,6 +681,18 @@ def launch_template(config:, launched_instances_name_prefix:)
       hosts = []
       terminate_spec = {}
       file_details = {}
+
+      # Export configuration's block of environment variables to running environment context
+      if (conf[:global, :"install-envvars"] != nil)
+        conf[:global, :"install-envvars"].each do |key, val|
+          if ENV.key?(key.to_s)
+            say "WARNING: Clashing value coming from Environment (priority) and configuration for key: '#{key.to_s}'"
```

Comment:
maybe better something along the lines of 

> WARNING: value from environment variable #{key.to_s} takes precedence over value in 'install-envvars' configuration

btw perhaps call it `install-env`, should be clear enough, not a hard requirement on my end

Reply:
ok, if we don't want to break existing usage, maybe keep `envvars` string. pls check doc fixes though.

Reply:
Agree, leaving name of section untouched, improving WARNING msg.